pipeline_tools/input_utils.py

```diff
@@ -1,335 +1,168 @@
+                            metadata_files_dict.items())
+            )
+    return metadata_files
 
-def get_version_prefix(schema_version):
-    try:
-        return int(schema_version.split('.', 1)[0])
-    except ValueError:
-        raise ValueError('Invalid schema version')
 
-
-def get_metadata_to_process(manifest_files, dss_url, is_v5_or_higher, http_requests):
-    """Return the metadata json that we need to parse to set up pipeline inputs
+def _get_content_for_ss2_input_tsv(bundle_uuid, bundle_version, dss_url, http_requests):
+    """Gather the necessary metadata for the ss2 input tsv.
 
     Args:
-        manifest_files (dict): file metadata
-        dss_url (str): the url of the DCP Data Storage Service
-        is_v5_or_higher (bool): whether metadata is v5 or higher
-        http_requests (HttpRequests): the HttpRequests object to use
+        bundle_uuid (str): the bundle uuid.
+        bundle_version (str): the bundle version.
+        dss_url (str): the url for the DCP Data Storage Service.
+        http_requests (HttpRequests): the HttpRequests object to use.
 
     Returns:
-        Tuple of three items: inputs metadata dict, sample metadata dict, schema version string
+        tuple: tuple of three strings; url for fastq 1, url for fastq 2, sample id
 
     Raises:
-        requests.HTTPError: for 4xx errors or 5xx errors beyond timeout
+        requests.HTTPError: on 4xx errors or 5xx errors beyond the timeout
     """
-    sequencing_protocol_id = None
-    if not is_v5_or_higher:
-        schema_version = '4.x'
-        inputs_metadata_file_uuid = dcp_utils.get_file_uuid(manifest_files, 'assay.json')
-        inputs_metadata_json = dcp_utils.get_file_by_uuid(inputs_metadata_file_uuid, dss_url, http_requests)
-        sample_id_file_json = inputs_metadata_json
-    else:
-        sample_id_file_uuid = dcp_utils.get_file_uuid(manifest_files, 'links.json')
-        inputs_metadata_file_uuid = dcp_utils.get_file_uuid(manifest_files, 'file.json')
-        inputs_metadata_json = dcp_utils.get_file_by_uuid(inputs_metadata_file_uuid, dss_url, http_requests)
-        schema_version = detect_schema_version(inputs_metadata_json)
-        sample_id_file_json = dcp_utils.get_file_by_uuid(sample_id_file_uuid, dss_url, http_requests)
-
-        version_prefix = get_version_prefix(schema_version)
-        if version_prefix >= 6:
-            protocol_uuid = dcp_utils.get_file_uuid(manifest_files, 'protocol.json')
-            protocol_json = dcp_utils.get_file_by_uuid(protocol_uuid, dss_url, http_requests)
-            sequencing_protocol = [protocol for protocol in protocol_json['protocols'] if 'sequencing_protocol' in protocol['content']['describedBy']]
-            sequencing_protocol_id = sequencing_protocol[0]['hca_ingest']['document_id']
 
-    return inputs_metadata_json, sample_id_file_json, schema_version, sequencing_protocol_id
+    print("Getting bundle manifest for id {0}, version {1}".format(bundle_uuid, bundle_version))
```

Comment:
If there is no specific reason to use `print`, I prefer to place all prints with logging throughout the pipeline-tools, any objections?